soroban-env-host/src/native_contract/token/contract.rs

```diff
@@ -378,8 +362,8 @@ impl TokenTrait for Token {
         Ok(())
     }
 
-    fn decimals(e: &Host) -> Result<u32, HostError> {
-        read_decimal(&e)
+    fn decimals(_e: &Host) -> Result<u32, HostError> {
+        Ok(7)
```

Comment:
We could still allow this to be configurable? The stellar network doesn't have any explicit notion of decimals for assets.

Reply:
Would you make this dynamically configurable (i.e. via a method) or configurable only during the token creation? If former, then it's a bit of the interface bloat. If latter, then I'm not sure how to make that safe (currently we rely on the well-defined Asset->metadata mapping process). I agree this sounds reasonable on paper, but I'm not sure what would be the proper implementation...

Reply:
Yeah I think we could, but I'm still not convinced it's a good idea. Either way, this can be done in a separate PR.